src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -25,6 +25,16 @@ class DateTimeNormalizer implements NormalizerInterface, DenormalizerInterface,
     const FORMAT_KEY = 'datetime_format';
     const TIMEZONE_KEY = 'datetime_timezone';
 
+    /**
+     * While denormalizing, we try to cast strings into real DateTime instances
+     * causing exception to be thrown if the string is not well formatted.
+     *
+     * Setting this tag to true will allow you to retrieve real instance of DateTime
+     * if the string is well formatted but will avoid exception if the string is not
+     * well formatted.
```

Comment:
proposal:
When setting this option to true, denormalization will ignore invalid dates and leave them as strings. This can be useful to replace exceptions thrown by the serializer by more contextual exceptions thrown by a later validation step.

Reply:
You mean, create a new Exception class which will be more specific than the NotNormalizableValueException ?

Reply:
By the way, in a world where this PR would have been approved, your proposal for this comment is much better than mine ^^